src/config.ts

```diff
@@ -44,6 +44,8 @@ export const SPID_LEVELS = {
   SpidL3: "https://www.spid.gov.it/SpidL3"
 };
 
+export const ENABLED_SPID_LEVELS: ReadonlyArray<string> = ["SpidL2", "SpidL3"];
```

Comment:
this must be a configuration that comes from io-backend someway

Reply:
Fixed in 92a5f1f